core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/GraphicElementFactory.java

```diff
@@ -94,7 +99,8 @@
       GraphicElementFactory.class + "_REQUEST_IS_COMPONENT_MAIN_PAGE";
   private final static String REQUEST_EXTERNAL_STYLESHEET =
       GraphicElementFactory.class + "_REQUEST_EXTERNAL_STYLESHEET";
-  private final static String defaultLook = "org.silverpeas.util.viewGenerator.settings.Initial";
+  private final static String defaultLook =
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "defaultLook" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1068)